src/main/java/io/appium/java_client/serverevents/CommandEvent.java

```diff
@@ -0,0 +1,13 @@
+package io.appium.java_client.serverevents;
+
+public class CommandEvent {
+    public final String name;
+    public final Long startTimestamp;
+    public final Long endTimestamp;
+
+    public CommandEvent(String name, Long startTimestamp, Long endTimestamp) {
```

Comment:
can we use long instead of Long?

Reply:
why I try that I am unable to decompose results into `List<long>` because list cannot take a primitive type.

Reply:
n/m seems to work in the definition anyway.

Reply:
have you heard about autoboxing in Java? ;)

Reply:
remember, i am not the world's best java person.

Reply:
not yet, not yet :)